includes/course-video/blocks/class-sensei-course-video-blocks-youtube-extension.php

```diff
@@ -15,83 +15,57 @@
  *
  * @since 4.0.0
  */
-class Sensei_Course_Video_Blocks_Youtube_Extension {
+class Sensei_Course_Video_Blocks_Youtube_Extension extends Sensei_Course_Video_Blocks_Embed_Extension {
 	/**
 	 * Instance of class.
 	 *
 	 * @var self
 	 */
 	private static $instance;
 
-	/**
-	 * Manages Video course-related settings.
-	 *
-	 * @var Sensei_Course_Video_Settings
-	 */
-	private $settings;
-
 	/**
 	 * Returns an instance of the class.
 	 *
-	 * @param Sensei_Course_Video_Settings $settings
-	 *
-	 * @return Sensei_Course_Video_Blocks_Youtube_Extension
+	 * @return static
 	 */
-	public static function instance( Sensei_Course_Video_Settings $settings ) {
+	public static function instance() {
 		if ( self::$instance ) {
 			return self::$instance;
 		}
 
-		self::$instance = new self( $settings );
+		self::$instance = new static();
```

Comment:
Just wondering if this should be `new self();`? It would be consistent with how the Vimeo extension is doing it.